pandas/tests/frame/apply/test_frame_transform.py

```diff
@@ -0,0 +1,72 @@
+import operator
+
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+from pandas.tests.frame.common import zip_frames
+
+
+def test_agg_transform(axis, float_frame):
+    other_axis = 1 if axis in {0, "index"} else 0
+
+    with np.errstate(all="ignore"):
+
+        f_abs = np.abs(float_frame)
+        f_sqrt = np.sqrt(float_frame)
+
+        # ufunc
+        result = float_frame.transform(np.sqrt, axis=axis)
+        expected = f_sqrt.copy()
+        tm.assert_frame_equal(result, expected)
+
+        result = float_frame.transform(np.sqrt, axis=axis)
+        tm.assert_frame_equal(result, expected)
+
+        # list-like
```

Comment:
same comments in other PR, ideally these cases are separate tests (or parameterized but that might be harder), (but can certainly be a followon PR)